client/assets/javascripts/modules/googleAnalyticsEvents.js

```diff
@@ -0,0 +1,29 @@
+class GoogleAnalyticsEvents {
```

Comment:
Worth checking that ES6 class semantics work on all our target browsers and if we need to polyfill for older browsers (IE-old for example).

Reply:
I've dropped in `"@babel/plugin-transform-classes"` to take care of older IE versions. The only exception was where a call extends a built-in type which we aren't doing so we should be good but I'll run the browserstack tests to be sure.